src/index.html

```diff
@@ -9,6 +9,33 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a class="header__logo" href="#">
+        <img
+          src="./images/logo.png"
+          alt="logo"
+        >
+      </a>
+      <nav class="header__nav nav">
+        <ul class="nav__list">
+          <li class="nav__item"><a class="nav__link is-active" href="#">Apple</a></li>
```

Comment:
```suggestion
          <li class="nav__item">
            <a class="nav__link is-active" href="#">
              Apple
            </a>
          </li>
```

Same as in the previous comments, apply it to all similar cases
Check this doc - https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules#html-content